.rubocop.yml

```diff
@@ -10,6 +10,7 @@ AllCops:
 Style/MethodCallWithArgsParentheses:
   Exclude:
     - '**/Gemfile'
+    - 'test/**/*'
```

Comment:
There were a number of failures on tests with DSL syntax such as:

```
test/shopify_app/controller_concerns/login_protection_test.rb:472:7: C: Style/MethodCallWithArgsParentheses: Use parentheses for method calls with arguments.
      assert_equal "{\"myshopifyUrl\":\"https://#{shop_domain}\",\"url\":\"#{example_url}\"}", ...
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
test/shopify_app/controller_concerns/login_protection_test.rb:480:9: C: Style/MethodCallWithArgsParentheses: Use parentheses for method calls with arguments.
        get '/' => 'login_protection#index'
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Since these typically don't have parenthesis, I disabled the rule for the whole `test` directory, although there is *some* risk that the `test` directory could contain other code with actual missing parenthesis.